frontend/website/src/pages/LeaderboardPage.re

```diff
@@ -0,0 +1,218 @@
+         )}
+      </p>
+      <p className=Theme.Body.basic>
+        {React.string(
+           "*Testnet Points (abbreviated 'pts') are designed solely to track contributions \
+           to the Testnet and Testnet Points have no cash or other monetary value. \
+           Testnet Points are not transferable and are not redeemable or exchangeable \
+           for any cryptocurrency or digital assets. We may at any time amend or eliminate Testnet Points.",
+         )}
+      </p>
+    </div>;
+  };
+};
+
+[@react.component]
+let make = () => {
+  <Page title="Testnet Leaderboard">
+    <Wrapped>
+      <div className=Styles.page>
+        <div className=Styles.heroRow>
+          <div className=Styles.heroLeft>
+            <h1 className=Theme.H1.basic>
+              {React.string("Testnet Leaderboard")}
+            </h1>
+            <Spacer height=4.3 />
+            <StatisticsRow />
+            <HeroText />
+          </div>
+          <div className=Styles.heroRight>
+            <div className=Styles.buttonRow>
+              <Button
+                link=""
+                label="Current Challenges"
+                bgColor=Theme.Colors.jungle
+                bgColorHover=Theme.Colors.clover
+              />
+              <Spacer width=2.0 />
+              <Button
+                link=""
+                label="Genesis Program"
+                bgColor=Theme.Colors.jungle
+                bgColorHover=Theme.Colors.clover
+              />
+            </div>
+            <Spacer height=4.8 />
+            <div className=Styles.heroLinks>
+              <div className=Styles.flexColumn>
+                <Next.Link href="">
+                  <a className=Theme.Link.basic>
+                    <Svg
+                      link="/static/img/Icon.Link.svg"
+                      dims=(1.0, 1.0)
+                      className=Styles.icon
+                      alt="an arrow pointing to the right with a square around it"
+                    />
+                    {React.string("Leaderboard FAQ")}
+                  </a>
+                </Next.Link>
+                <Next.Link href="">
+                  <a className=Theme.Link.basic>
+                    <Svg
+                      link="/static/img/Icon.Link.svg"
+                      dims=(0.9425, 0.8725)
+                      className=Styles.icon
+                      alt="an arrow pointing to the right with a square around it"
+                    />
+                    {React.string("Discord #Leaderboard Channel")}
+                  </a>
+                </Next.Link>
+                <span className=Theme.Link.basic>
+                  <Svg
+                    link="/static/img/Icon.Info.svg"
+                    className=Styles.icon
+                    dims=(1.0, 1.0)
+                    alt="a undercase letter i inside a blue circle"
+                  />
+                  {React.string("Last manually updated May 1 2020")}
```

Comment:
Can we interpolate a a formatted date here? Even if we're not pulling it from the backend just so it's easier to plug into once we have the data setup.

Reply:
what do you mean interpolate? like "May" + "1" + "2020"?